RetrospectiveExtension.Frontend/components/__tests__/feedbackBoard.test.tsx

```diff
@@ -1,3 +1,119 @@
+import React from 'react';
+import { IFeedbackBoardDocument } from '../../interfaces/feedback';
+import { shallow } from 'enzyme';
+import { mocked } from 'jest-mock';
+import FeedbackBoard, { FeedbackBoardProps } from '../../components/feedbackBoard';
+import { testColumnProps } from '../__mocks__/mocked_components/mockedFeedbackColumn';
+import { IdentityRef } from 'azure-devops-extension-api/WebApi';
+
+jest.mock('../feedbackBoardMetadataForm', () => mocked({}));
+jest.mock('azure-devops-extension-api/Work/WorkClient', () => {
+  const getTeamIterationsMock = () => {
```

Comment:
These mocks are copied from `feedbackBoardContainer.test.tsx`. I don't know if they're general enough to be moved to the `__mocks__` folder. It might also be possible to use a simplified version in these tests.

Reply:
If this chunk is a copy of the other, I think that's a valid thing to move to `__mocks__` folder so there's less duplicate code. It's probably okay to remove any bit that's not explicitly called, but that may be worth doing a local test or two to be sure.